tests/command.dev.test.js

```diff
@@ -185,7 +185,7 @@ testMatrix.forEach(({ args }) => {
         })
         .withFunction({
           path: 'echo.js',
-          handler: async (event, context) => {
+          handler: (event, context) => {
```

Comment:
I think functions should return a promise when not using the callback pattern.
We're using `lambda-local` to invoke functions locally - if the tests pass I can only assume it handles it.

Reply:
This makes sense, thanks for the additional information/

The related ESLint rule is `require-await`. Generally speaking, this might be a good rule to decrease the number of `async`/`await` in our code, and only make code asynchronous that needs to be.

Is there a way we could still enable this rule (if you find it useful)? For example, one of:
  - Adding `eslint-disable-next-line` comment before each `handler`
  - Adding `eslint-disable` comment for the whole file
  - Adding an `overrides` to disable this rule in all `*.test.js`
  - Relying on `lambda-local` behavior allowing sync functions

What do you think?

Reply:
I think the best approach will be to disable it for the file with a comment explaining the reason.

Reply:
Fixed!